protocol/communities/community.go

```diff
@@ -1363,6 +1363,7 @@ func (o *Community) AddTokenPermission(permission *protobuf.CommunityTokenPermis
 	}
 
 	o.config.CommunityDescription.TokenPermissions[permission.Id] = permission
+	o.updateEncrypted()
```

Comment:
If I understood correctly, if you'll do updateEcrypted, `UpdateCommunityEncryption` will always return nil due to
```
if community.Encrypted() == isEncrypted {
		return nil
	}
```

Reply:
ok, I see, it was removed

Reply:
What about admin events? What if the admin will add member permission? Add/Delete/Update token CommunityEvent must include encryption information in this case